src/java.base/share/classes/java/nio/charset/Charset.java

```diff
@@ -628,7 +628,8 @@ public SortedMap<String,Charset> run() {
             });
     }
 
-    private static volatile Charset defaultCharset;
+    @Stable
+    private static Charset defaultCharset;
```

Comment:
Style wise, I think the annotation is added between the private and static modifiers in other places.

Reply:
Fixed.